src/System.IO.Compression/tests/DeflateStreamTests.netstandard1.7.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public class DeflateStreamTests_NS17 : DeflateStreamTests
```

Comment:
You are right, I only did it because I wanted to use ManualSyncMemoryStream class which is  nested under DeflateStreamTests, so one thing I can do is just move that class out of the test class and mark it as internal.